src/Pdf/CakePdf.php

```diff
@@ -738,6 +752,40 @@ public function template($template = false, $layout = null)
     }
 
     /**
+     * Template path
+     *
+     * @param mixed $templatePath The path of the template to use
+     * @return mixed
+     */
+    public function templatePath($templatePath = false)
```

Comment:
I wonder if null would be a better default null value here. Then docblock: string|null